server/utils.js

```diff
@@ -46,6 +50,33 @@ const utils = {
     const nums = new Array(n).fill(0).map((a, i) => new Uint8Array(otps.slice(i * 4, (i + 1) * 4))).map(utils.decodeOtp)
     return nums.map(i => i.toString().padStart(6, '0'))
   },
+
+  parseTx: (tx) => {
+    const txId = tx?.tx
```

Comment:
this might be truffle only

Reply:
Have removed `parseTx` for now as it was not being used.